web/templates/page/index.html.eex

```diff
@@ -17,14 +36,33 @@
   <div class="pa3 bg-near-white b--black-10">
     <%= if @tip do %>
       <p class="pv2 ph2 ph3-m ph4-l f5 f4-l lh-copy"><%= markdown @tip.content %></p>
-      <a href="/tips/?tag=<%= tag = cond do
+      <a href="/tips?tag=<%= tag = cond do
```

Comment:
This feels strange 🤔

Ideally, we'd like as little code/logic in our templates as possible, could this be moved into a helper in the `View`?